pt_BR.ini

```diff
@@ -18,17 +18,17 @@ DownloadFromStore = Download do repositório de homebrew do PPSSPP
 
 [Game]
 Play = Jogar
-Game Settings = Configurações do jogo
+Game Settings = Configurações
```

Comment:
It should be "Configurações do jogo", because it is specific settings for the current game. It appears after clicking on Create Game Config.